public/app/plugins/panel/pmm-update/components/ConfirmUpdateModal/ConfirmUpdateModal.messages.ts

```diff
@@ -0,0 +1,13 @@
+export const Messages = {
+  title: 'PMM Upgrade',
+  version: (oldVersion: string, newVersion: string) =>
+    `Your PMM is currently in the version is ${oldVersion} and you’re about to upgrade to the version ${newVersion}. `,
```

Comment:
typo

Reply:
Removed, but haven't received final wording yet, so it might change.